contrib/auth/apigen/oapi-tmpl/chi-interface.tmpl

```diff
@@ -0,0 +1,11 @@
+// chi-interface.tmpl based on https://github.com/deepmap/oapi-codegen/tree/master/pkg/codegen/templates
```

Comment:
Can we somehow re-use the same templates from lakeFS?  I think it is reasonable for contrib to use main.

Reply:
Trying